### ISSUE TITLE
[Warlock][Health Funnel] health cost bugg

### ISSUE THREAD

**Issue by axdelol** [dalaranwow/dalaran-wow#4332]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Cost 3 times more health than what spell says it should making it useless**: At health bar even if dialogue its ok it seems to costs every second what only should cost at the beginning of the casting, plus what should cost every second.

**it actually shoul cost mana and not health according to  wotlk.openwow.com website**: 

**Steps to reproduce the problem**:

1. Cast the spell health funnel in your minion
2. Check health cost at health bar, dialogue is ok, but you can see it cost every second what only should cost at the beginning of the spell plus what the health should cost every second, making it useless.
3. its a needed spell for afliction warlocks who their minions are should be the first objetives when plaaying with priests, shamans, paladin and druids couse the minion is what gives sustain to the warlock.

**http://wotlk.openwow.com/spell=47856**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**http://wotlk.openwow.com/spell=47856**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/239
